label_studio_ml/examples/yolo/docker-compose.yml

```diff
@@ -46,3 +46,5 @@ services:
       - "9090:9090"
     volumes:
       - "./data/server:/data"
+      - "./models:/app/models"
```

Comment:
@micaelakaplan could you please check if models and cache_dir are created outside of the docker in the folder where you are running the ML backend?

Reply:
They are not 😢